content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/10-migrate-vm-to-different-node.md

```diff
@@ -14,67 +14,134 @@ import InfoBox from 'shared/components/InfoBox';
 
 # Overview
 
-Live migration is a process in which a running virtual machine (VM) moves to another cluster node while the guest workload continues to run. During live migration, the VM remains accessible.
+Palette supports seamless virtual machine (VM) migration to another physical host in the cluster. This is known as *live migration*. During live migration, the VM and its memory, storage, and CPU resources are moved from one cluster compute node to another without any noticeable downtime. 
 
-Successful live migrations rely on appropriately configured storage and networking and must be enabled as a feature gate. Live migration is enabled by default in the ``feature-gates`` field of the kubevirt configuration file in the Spectro VM Dashboard pack. Refer to [Feature Gates](/vm-management#featuregates).
+Successful live migrations rely on appropriately configured storage and networking, and live migration must be enabled as a feature gate. Live migration is enabled by default in the ``feature-gates`` section of the KubeVirt configuration file that is part of the Spectro VM Dashboard pack. For more information, review the [Feature Gates](/vm-management#featuregates) section.
 
-Consider the following when migrating a VM node:
+Live migration is used with rolling Kubernetes upgrades and workload balancing. To avoid interrupting a VM when a node is placed into maintenance or upgraded, all VM instances require a ``LiveMigrate`` eviction strategy. By default, the ``spec.template.spec.evictionStrategy`` parameter is set to ``LiveMigrate`` in the KubeVirt configuration file. 
```

Comment:
```suggestion
Live migration is used with rolling Kubernetes upgrades and workload balancing. To avoid interrupting a VM when a node is placed into maintenance or upgraded, all VM instances require a ``LiveMigrate`` eviction strategy.
```